src/product/security.md

```diff
@@ -0,0 +1,58 @@
+FlowForge has multiple editions, and the security aspects depends on both the
+configuration of FlowForge as well as the application.
+
+## Application security
+
+Security on the application level will have the same properties for self-managed
+installs as well as FlowForge Cloud.
+
+### Authentication and authorization
+
+Authentication and session management is centrally managed in the FlowForge
+application. There's currently no support for SAML or SSO. There's no support
+for centralized management for creation of teams, users per team, nor their RBAC
+roles.
+
+## Configured properties
+
+Configuration of the FlowForge application influences security aspects of the
+system at large. Below 
+
+### FlowForge Cloud
+
+FlowForge Cloud is hosted on Amazon Web Services (AWS) in Ireland.
+
+#### Service endpoints
+
+Our cloud offering has multiple service domains. The corporate website is hosted
+at [flowforge.com](https://flowforge.com). The application is served at
+[app.flowforge.com](https://app.flowforge.com), while the Node-RED instances are
+served from \*.flowforge.cloud.
+
+### Data Protection
+
+Data protection has two components: while in-transit and at rest.
+
+#### Data in-transit
+
+Data in transit from external sources is protected with SSL/TLS. Encryption is
+terminated at the edge, both for HTTP and MQTT traffic.
+
+#### Data at rest
+
+FlowForge Cloud stores data in either our relational database or an object
+store. For both the database and the object storage there's currently no
+encryption at rest configured.
+
```

Comment:
Can't add an issue/epic here just yet as it's in the private CloudProject (https://github.com/flowforge/CloudProject/issues/79)